modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/public-navbar/index.css

```diff
@@ -1,3 +1,27 @@
+.show-menu {
+	opacity: 1;
+	transform: translateY(5px);
+	transition: cubic-bezier(0.215, 0.61, 0.355, 1) 0.3s;
+}
+
+.hiden-menu {
```

Comment:
Please, change class name ```hiden-menu``` to ```hidden-menu``` and modify this class in all files that its used.